src/subcommands/projects.ts

```diff
@@ -101,18 +108,26 @@ async function showProject(args: Args): Promise<void> {
   const api = args.token
     ? API.fromToken(args.token)
     : API.withTokenProvisioner(TokenProvisioner);
-  const [project, domains, pagedBuilds] = await Promise.all([
+  const [project, domains, pagedBuilds, databases, crons] = await Promise.all([
     api.getProject(args.project),
     api.getDomains(args.project),
     api.listDeployments(args.project),
+    api.getProjectDatabases(args.project),
+    api.getDeploymentCrons(args.project, "latest").catch((err) => {
+      // When the project does not have a production deployment, API returns deploymentNotFound
+      if (err instanceof APIError && err.code === "deploymentNotFound") {
+        return null;
+      }
+      throw err;
+    }),
```

Comment:
Any particular reason to handle this error case here instead of in `getDeploymentCrons`?

Reply:
Yes. I din't want to add a null in the signature only for an special usecase (using "latest" as deployment-id). What I've ended up doing is adding `getProjectCrons()`